features/salt_openscap_audit.feature

```diff
@@ -0,0 +1,58 @@
+# Copyright (c) 2017 SUSE LLC
+# Licensed under the terms of the MIT license.
+
+Feature: Use the openSCAP audit feature in SUSE Manager for a salt minion
+  In order to audit my salt minions
+  As an authorized user
+  I want to run openSCAP scans on them
+
+  Scenario: Schedule an audit job on the minion
+    Given I am on the Systems overview page of this "sle-minion"
+    And I disable IPv6 forwarding on all interfaces of this "sle-minion"
+    And I follow "Audit" in the content area
+    And I follow "Schedule" in the content area
+    When I enter "--profile Default" as "params"
+    And I enter "/usr/share/openscap/scap-yast2sec-xccdf.xml" as "path"
+    And I click on "Schedule"
+    Then I should see a "XCCDF scan has been scheduled" text
+    And I wait for "10" seconds
```

Comment:
@Bischoff  which action are we waiting here for?
In order to avoid the usage of sleep